packages/react/test/specs/components/Dropdown/Dropdown-test.tsx

```diff
@@ -323,15 +486,302 @@ describe('Dropdown', () => {
+        null,
+        expect.objectContaining({
+          value: items[0],
+        }),
+      )
+    })
+
+    it('is set by using Shift+Tab on highlighted item', () => {
+      const onSelectedChange = jest.fn()
+      const wrapper = mountWithProvider(
+        <Dropdown onSelectedChange={onSelectedChange} items={items} />,
+      )
+      const triggerButton = wrapper.find(`button.${Dropdown.slotClassNames.triggerButton}`)
+
+      triggerButton.simulate('click')
+      const itemsList = wrapper.find(`ul.${Dropdown.slotClassNames.itemsList}`)
+      itemsList
+        .simulate('keydown', { keyCode: keyboardKey.ArrowDown, key: 'ArrowDown' })
+        .simulate('keydown', { keyCode: keyboardKey.Tab, key: 'Tab', shiftKey: true })
+
+      expect(onSelectedChange).toHaveBeenCalledTimes(1)
+      expect(onSelectedChange).toHaveBeenCalledWith(
+        null,
+        expect.objectContaining({
+          value: items[0],
+        }),
+      )
+    })
+
+    it('is replaced when another item is selected', () => {
+      const itemSelectedIndex = 3
+      const onSelectedChange = jest.fn()
+      const wrapper = mountWithProvider(
+        <Dropdown onSelectedChange={onSelectedChange} items={items} />,
+      )
+      const triggerButton = wrapper.find(`button.${Dropdown.slotClassNames.triggerButton}`)
+
+      triggerButton.simulate('click')
+      const firstItem = wrapper.find(`li.${Dropdown.slotClassNames.item}`).at(1)
+      firstItem.simulate('click')
+      triggerButton.simulate('click')
+      const itemAtIndex = wrapper.find(`li.${Dropdown.slotClassNames.item}`).at(itemSelectedIndex)
+      itemAtIndex.simulate('click')
+
+      expect(onSelectedChange).toHaveBeenCalledTimes(2)
+      expect(onSelectedChange).toHaveBeenLastCalledWith(
+        null,
+        expect.objectContaining({
+          value: items[itemSelectedIndex],
+        }),
+      )
+    })
+
+    it('has multiple items at multiple selections if the multiple prop is supplied', () => {
+      const onSelectedChange = jest.fn()
+      const wrapper = mountWithProvider(
+        <Dropdown onSelectedChange={onSelectedChange} multiple items={items} />,
+      )
+      const indexesOfItemsSelected = [1, 3]
+      const triggerButton = wrapper.find(`button.${Dropdown.slotClassNames.triggerButton}`)
+
+      triggerButton.simulate('click')
+      const itemAtIndex1 = wrapper
+        .find(`li.${Dropdown.slotClassNames.item}`)
+        .at(indexesOfItemsSelected[0])
+      itemAtIndex1.simulate('click')
+      triggerButton.simulate('click')
+      const itemAtIndex2 = wrapper
+        .find(`li.${Dropdown.slotClassNames.item}`)
+        .at(indexesOfItemsSelected[1])
+      itemAtIndex2.simulate('click')
+
+      expect(onSelectedChange).toHaveBeenCalledTimes(2)
+      expect(onSelectedChange).toHaveBeenLastCalledWith(
+        null,
+        expect.objectContaining({
+          value: [items[indexesOfItemsSelected[0]], items[indexesOfItemsSelected[1] + 1]],
```

Comment:
```suggestion
          value: [items[1], items[4]],
```

Can we use constant values there? It's very hard to read...

Reply:
https://github.com/stardust-ui/react/pull/65#discussion_r208722857